src/openqaoa-core/problems/vehiclerouting.py

```diff
@@ -0,0 +1,425 @@
+import networkx as nx
+from docplex.mp.model import Model
+from typing import Union
+import numpy as np
+import itertools
+import copy
+import matplotlib.pyplot as plt
+from matplotlib import colormaps
+
+from .problem import Problem
+from .converters import FromDocplex2IsingModel
+from .qubo import QUBO
+
+
+class VRP(Problem):
+    """
+    Creates an instance of the vehicle routing problem (VRP).
+    https://en.wikipedia.org/wiki/Vehicle_routing_problem
+
+    Parameters
+    ----------
+    G: networkx.Graph
+        Networkx graph of the problem
+    pos: list[list]
+        position x, y of each node
+    n_vehicles: int
+        the number of vehicles used in the solution
+    depot: int
+        the node where all the vehicles leave for and return after
+    subtours: list[list]
+        if -1 (Default value): All the possible subtours are added to the constraints. Avoid it for large instances.
+        if there are subtours that want be avoided in the solution, e.g, a 8 nodes
+        VRP with an optimal solution showing subtour between nodes 4, 5, and 8 can be
+        avoided introducing the constraint subtours=[[4,5,8]]. To additional information
+        about subtours refer to https://de.wikipedia.org/wiki/Datei:TSP_short_cycles.png
+    all_subtours:
+    penalty: float
+        Penalty for the constraints. If the method is 'unbalanced' three values are needed,
+        one for the equality constraints and two for the inequality constraints.
+    method: str
+        Two available methods for the inequality constraints ["slack", "unbalanced"]
+        For 'unblanced' see https://arxiv.org/abs/2211.13914
+    Returns
+    -------
+        An instance of the VRP problem.
+    """
+
+    __name__ = "vehicle_routing"
+
+    def __init__(
+        self,
+        G,
+        pos,
+        n_vehicles,
+        depot: int = 0,
+        subtours: list = -1,
```

Comment:
Did the default = `None` not work for this argument?

Reply:
Yes, but I want for the default argument to do all the subtours and subtours=[] for not adding any subtour constraint. subtours = None has the problem if one wants to convert the qubo into a dict using `todict` because it does not save the key if the value is None. (I just found it when testing).